### PR TITLE
Update copyright

### PR DESCRIPTION
**Pull request by rickwierenga**

### CODE REVIEW

File: source/_footer.slim [hunk 1]
```diff
@@ -2,7 +2,7 @@ footer
     .container
         .contact.mb-2.d-flex.justify-content-between
             .left
-                span &copy;2018-2019 iina.io
+                span &copy;2018-2020 iina.io
```
Comment (lucsoft) [inline source/_footer.slim:5]:
```suggestion
                span &copy;2018-2021 iina.io
```

**Review by lucsoft — APPROVED** [review]

**Review by vinayakkulkarni — CHANGES_REQUESTED** [review]
https://github.com/iina/iina-website/pull/13

I still think above PR is better approach than hardcoding the date.

**Comment by alextechtai** [comment]
👍🏻 Bump :) It's almost 2022!

File: source/_footer.slim [hunk 2]
```diff
@@ -2,7 +2,7 @@ footer
     .container
         .contact.mb-2.d-flex.justify-content-between
             .left
-                span &copy;2018-2019 iina.io
+                span &copy;2018-2021 iina.io
```
Comment (lucsoft) [inline source/_footer.slim:5]:
```suggestion
                span &copy;2018-2022 iina.io
```

**Review by lucsoft — CHANGES_REQUESTED** [review]
See us next year 👋

File: source/_footer.slim (hunk 2 above)
Comment (vinayakkulkarni) [reply source/_footer.slim:5]: why hardcode ?

**Comment by vinayakkulkarni** [comment]
Should be closed via – https://github.com/iina/iina-website/pull/13

File: source/_footer.slim (hunk 2 above)
Comment (rickwierenga) [reply source/_footer.slim:5]: Yes, let's go with https://github.com/iina/iina-website/pull/13 as you (@vinayakkulkarni) commented earlier.

**Pull request closed without merging**